templates/add-persistence-attribute-SEND_PHONE_MESSAGE/code.js

```diff
@@ -32,31 +30,27 @@ exports.onExecuteSendPhoneMessage = async (event, api) => {
     const metadataDefaultValue = event.secrets.METADATA_DEFAULT_VALUE;
 
     if (!event.secrets.CLIENT_ID) {
-        console.log('missing event.secrets.CLIENT_ID');
-        return;
+        return api.access.deny('missing client id');
```

Comment:
You also need to revert these. The `SendPhoneMessage` trigger does not have `api.access.deny` https://auth0.com/docs/customize/actions/flows-and-triggers/send-phone-message-flow